hail_scripts/v01/utils/elasticsearch_utils.py

```diff
@@ -174,17 +186,25 @@ def parse_vds_schema(vds_variant_schema_fields, current_parent=()):
             schema - for example: ("va", "info", "AC"), and values are hail field types as strings -
             for example "Array[String]".
     """
+    if not current_parent:
+        current_parent = []
+
     field_path_to_field_type_map = {}
     for field in vds_variant_schema_fields:
         field_name = field.name
         field_type = str(field.typ)
-        if field_type.startswith("Array") and ".".join(current_parent) not in ["v", "va", "va.info"]:
-            raise ValueError(".".join(current_parent)+".%(field_name)s (%(field_type)s): nested array types not yet implemented." % locals())
-        if field_type.startswith("Struct"):
-            child_schema = parse_vds_schema(field.typ.fields, current_parent + [field_name])
-            field_path_to_field_type_map.update(child_schema)
-        else:
-            field_path_to_field_type_map[tuple(current_parent + [field_name])] = field_type
+        try:
+            if field_type.startswith("Array[Struct{"): #and ".".join(current_parent) not in ["v", "va", "va.info"]:
+                nested_schema = parse_vds_schema(field.typ.element_type.fields, [])
+                field_path_to_field_type_map[tuple(current_parent + [field_name])] = nested_schema
+            elif field_type.startswith("Struct"):
+                struct_schema = parse_vds_schema(field.typ.fields, current_parent + [field_name])
+                field_path_to_field_type_map.update(struct_schema)
+            else:
+                field_path_to_field_type_map[tuple(current_parent + [field_name])] = field_type
+        except Exception as e:
+            traceback.print_exc()
+            raise ValueError("Error processing field: %s[%s]: %s" % (field_name, field_type, e))
```

Comment:
Maybe include `current_parent` with `field_name` in case a nested field has the same name as a top-level field.